src/driver/AmdCompiler.cpp

```diff
@@ -522,15 +535,62 @@ bool AMDGPUCompiler::CompileToLLVMBitcode(Data* input, Data* output, const std::
 
   File* bcFile = ToOutputFile(output, CompilerTempDir());
 
-  args.push_back("-o"); args.push_back(bcFile->Name().c_str());
+  args.push_back("-o");
+  args.push_back(bcFile->Name().c_str());
 
   for (const std::string& s : options) {
     args.push_back(s.c_str());
   }
 
-  bool res = InvokeDriver(args);
-  if (res) { output->ReadOutputFile(bcFile); }
-  return res;
+  if (inprocess) {
+    std::unique_ptr<Driver> driver(new Driver(llvmBin + "/clang", "amdgcn-amd-amdhsa-opencl", diags));
```

Comment:
Driver's constructor doesn't check the executable existence, thus empty string might be passed, I double checked it.